### ISSUE TITLE
additional_copy_path breaks jinja includes

### ISSUE THREAD

**Issue by boernd** [neillturner/kitchen-ansible#203]
As soon as I add the additional_copy_path property in one of my suites, includes in jinja templates throw errors. 

Suite example:
```
...
suites:
name: test
    - provisioner:
      playbook: playbooks/test.yml
      additional_copy_path: [ playbooks/test2.yml ]
      hosts: test
    driver_config:
      hostname: test
      instance_name: test
...
```

The test.yml playbook includes a test role which templates a file:
`{% include "testfile" %}`

`kitchen setup test` causes the following error:
```
      TASK [test : template test] ****************************************************
       fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "IOError: [Errno 20] Not a directory: '/tmp/kitchen/test2.yml/testfile'"}
```
Using recursive_additional_copy_path instead resolves the issue (/tmp/kitchen/test2.yml is not prefixed).

**Comment by neillturner** [comment]
get some debug output by adding -l debug to the command and then we can look at both cases to see what's different.

**Comment by boernd** [comment]
Thx for the hint. The debug output shows that the additional playbook configured in additional_copy_path is appended to the ANSIBLE_ROLES_PATH env var in the start command of the play:

`[SSH] kitchen@localhost<{:user_known_hosts_file=>"/dev/null", :paranoid=>false, :port=>32789, :compression=>true, :compression_level=>6, :keepalive=>true, :keepalive_interval=>60, :timeout=>15, :keys_only=>true, :keys=>["/home/boernd/sandbox/ansible/addcopypath/.kitchen/docker_id_rsa"], :auth_methods=>["publickey"], :password=>"kitchen", :user=>"kitchen"}> (ANSIBLE_ROLES_PATH=/tmp/kitchen/roles:/tmp/kitchen/test2.yml sudo -E -s  ansible-playbook -i /tmp/kitchen/host  -c local -M /tmp/kitchen/modules         /tmp/kitchen/test.yml)`

**Comment by neillturner** [comment]
I will create a new parameter  additional_copy_role_path   true/false
which will cause the additional copy path to be added to the ANSIBLE_ROLES_PATH

**Comment by neillturner** [comment]
fixed in 0.45.4
